data/campaigns/Legend_of_Wesmere/scenarios/chapter3/11_Battle_of_the_Book.cfg

```diff
@@ -284,6 +284,7 @@
 #else
             bonus=yes
 #endif
+            carryover_percentage=40
```

Comment:
I think the carryover here should be 80 and not 40. Do you even know what the next scenario is?
2 cutscenes and then the large-scale orc vs elf & human battle. I dont think players would even approve of this...

Reply:
80 is the default. Yes, I saw that the next scenario is Human Alliance. The question is what the intention is for converting all the campaigns - including this one - to use 40% by default. Why make it 80 if all the others are 40?